src/cf.data.pre

```diff
@@ -473,6 +473,26 @@ DOC_START
 	CAP_IPC_LOCK capability, or equivalent.
 DOC_END
 
+NAME: hopeless_kid_revival_delay
+COMMENT: time-units
+TYPE: time_t
+LOC: Config.hopelessKidRevivalDelay
+DEFAULT: 60 minutes
```

Comment:
parseTimeUnits() does not accept plurals and the definition of T_MINUTE_STR is "minute". You may want to use "1 hour" though instead of "60 minute"